nx-dev/nx-dev/pages/[version]/[flavor]/[...segments].tsx

```diff
@@ -31,6 +32,12 @@ export function Documentation({
   flavor,
   flavors,
 }: DocumentationProps) {
+  const router = useRouter();
+
+  if (router.isFallback) {
+    return <div>Nope</div>;
```

Comment:
Should this be a 404?

Reply:
Oops,  need to remove it since it was just for testing. We alway redirect so this should never be hit.